src/beat_link_trigger/show_cues.clj

```diff
@@ -1541,7 +1541,7 @@
                                     new-cue             (merge cue {:uuid    uuid
                                                                     :start   start
                                                                     :end     end
-                                                                    :hue     (assign-cue-hue context)
+                                                                    :hue     (get-in cue [:hue])
```

Comment:
For backwards compatibility with files that were created before this change, it would be safer for this to be `(or (get-in cue [:hue]) (assign-cue-hue context))` This way, newer files which had saved the hue information would preserve it, while older files which had the `:hue` element removed from the library would continue to assign a distinct hue for such cues.